.github/workflows/spread-tests.yaml

```diff
@@ -85,6 +85,8 @@ jobs:
       run: |
           rm -rf "${{ github.workspace }}"
           mkdir "${{ github.workspace }}"
+          # Make sure there are not spread processes running
+          killall spread || true
```

Comment:
Can you please do something slightly different instead:
```sh
killall -s SIGQUIT spread
```

I would like to see what spread is doing. I don't quite believe that PS7 work is affecting spread. I suspect something else is going on and this is just the symptom. 

Separately to this, could we run spread with tee to some log or cloud observability service. I would like to see the backtraces this produces.